resources/views/common/breadcrumbs.html.twig

```diff
@@ -1,19 +1,19 @@
 <nav aria-label="breadcrumb" data-c-breadcrumbs data-c-colour="{{ font_color ?: 'white' }}">
-	<ol>
-		{% for key, url in breadcrumbs %}
+	<ol id="{{ breadcrumbs_id }}">
```

Comment:
From what I can see this is not set on most pages, resulting in an empty id tag which is technically invalid HTML.